src/Symfony/Component/Security/Core/Exception/AuthenticationException.php

```diff
@@ -19,7 +19,7 @@
  * @author Fabien Potencier <fabien@symfony.com>
  * @author Alexander <iam.asm89@gmail.com>
  */
-class AuthenticationException extends \RuntimeException implements \Serializable
+class AuthenticationException extends \RuntimeException implements \Serializable, ExceptionInterface
```

Comment:
This is OK but it's a fix for only one of the sub-exceptions ;)

IMHO it would be better to make all exceptions in `\Symfony\Component\Security\Core\Exception\` namespace that extend `\RuntimeException` extend `\Symfony\Component\Security\Core\Exception\RuntimeException` instead.

Reply:
I agree